rosdep/base.yaml

```diff
@@ -2141,6 +2141,12 @@ libceres-dev:
   rhel:
     '7': [ceres-solver-devel]
   ubuntu: [libceres-dev]
+libcgroup-dev:
+  debian: [libcgroup-dev]
+  fedora: [libcgroup-devel]
+  gentoo: [dev-libs/libcgroup]
+  openembedded: [libcgroup@openembedded-core]
```

Comment:
I'm not familiar with openembedded, but there seems to be a recipe here: https://layers.openembedded.org/layerindex/recipe/576/